plugins/modules/source_control/gitlab/gitlab_branch.py

```diff
@@ -150,9 +141,6 @@ def main():
     ref_branch = module.params['ref_branch']
     state = module.params['state']
 
-    if not HAS_GITLAB_PACKAGE:
-        module.fail_json(msg=missing_required_lib("python-gitlab"), exception=GITLAB_IMP_ERR)
-
     gitlab_version = gitlab.__version__
```

Comment:
(Looks like I forgot to add this one first...)

This will crash when `gitlab = None`.